pkg/inject/inject.go

```diff
@@ -150,22 +137,27 @@ func (conf *ResourceConfig) YamlMarshalObj() ([]byte, error) {
 
 // ParseMetaAndYaml fills conf fields with both the metatada and the workload contents
 func (conf *ResourceConfig) ParseMetaAndYaml(bytes []byte) (*Report, error) {
-	if _, err := conf.ParseMeta(bytes); err != nil {
+	if _, err := conf.ParseMeta(bytes, ""); err != nil {
 		return nil, err
 	}
 	r := newReport(conf)
-	return &r, conf.parse(bytes)
+	return &r, conf.Parse(bytes)
 }
 
 // ParseMeta extracts metadata from bytes.
-// It returns false if the workload's payload is empty
-func (conf *ResourceConfig) ParseMeta(bytes []byte) (bool, error) {
+// If bytes doesn't contain the namespace (webhook case) then it must be
+// provided in the second argument.
+// It returns false if the workload's payload is empty.
+func (conf *ResourceConfig) ParseMeta(bytes []byte, ns string) (bool, error) {
```

Comment:
Per _slack_ discussion, the second argument isn't necessary since it's not being used anywhere.